tutorials/receiving-money.md

```diff
@@ -0,0 +1,275 @@
+
+### Common Errors
+
+- The most common mistake people make is not including the Authorization Header. All API requests on Paystack are authenticated with your secret key. Format is `Authorization: "Bearer YOUR_SECRET_KEY"`  Don’t leave out the `Bearer` keyword
+- `amount` value is in kobo. Always remember to multiply the Naira value of the amount by 100. So if you want to charge N100, you will pass amount as `100*100 = 10000`.
+- Calls to the Paystack API **should not be made from your frontend**, only from your server. This is because your secret key should never be exposed publicly. It can be used to access sensitive information on your Paystack account. The public key can only be used to initialize transactions on the Javascript Paystack Inline payment method.
+- Ensure to call the verify function from your callback function (for frontend implementation) or your callback URL/route (for standard redirect implementation). Only when control has been passed to your callback can you be certain that the user has completed payment. If you try to do a long pool to the verify endpoint, you will keep getting abandoned status until the user has completed the payment.
+
+### Use Case
+
+- An online store
+
+## Payment Channels
+
+Paystack provides various payment channels to allow merchant receive payment from customers.
+
+Currently we have
+
+- Pay with Card
+- Pay with Bank - this allows users complete payments using their Nigerian bank accounts
+- Pay with USSD
+- Pay with Ghana Mobile Money.
+
+The purpose of multiple channels is to allow the users complete payments using any of the channels in case any of them is down.
+
+You can enable or disable any of these Payment channels on your **Dashboard >> Settings >> Preferences**.
+
+## Best Practices
+
+* If you intend to charge recurring payments, it's best to enable only card payments, as other payment channels don't currently support reccuring billing.
+* If your transaction amounts are always high, it's best to only allow card payment as other channels may not be able to processs very high transaction amounts. Their failures may affect your payment success rate numbers.  
+
+## Split Payments
+
+###Why?
+
+Paystack has a Split Payments feature that allows you to split a transaction between your main Paystack account and a sub-account using a predefined split percentage, and Paystack automatically settles both the bank account of the main account and the sub-account.
+
+For example, if your business has a multiple products and you want a percentage of payments for each product to go into a account for that product while the rest goes into a central account, you can create a sub-account for each of the products, and include the sub-account code for each product in the transaction for that product.
```

Comment:
For example, if your business has multiple products and you want a percentage of payments for each product to go into an account for that product while the rest goes into a central account, you can create a sub-account for each of the products, and include the sub-account code for each product in the transaction for that product.

**_grammatical observation_**